### PR TITLE
rename CreateBatchFeature with CreateFeature

### PR DESCRIPTION
**Pull request by lianxmfor**
this PR is renamed `CreateBatchFeature` to `CreateFeature` and makes it support create stream feature.

relate #820

### CODE REVIEW

File: pkg/oomstore/feature_test.go [hunk 1]
```diff
@@ -78,12 +75,9 @@ func TestCreateBatchFeature(t *testing.T) {
...
 			}
-			_, err := store.CreateBatchFeature(ctx, tc.opt)
-			if tc.expectError {
-				assert.Error(t, err, fmt.Errorf("expected batch feature group, got %s feature group", tc.group.Category))
-			} else {
-				assert.NoError(t, err)
-			}
+
+			_, err := store.CreateFeature(ctx, tc.opt)
+			assert.NoError(t, err)
```
Comment (jinghancc) [inline pkg/oomstore/feature_test.go:80]: add test case for non-batch/stream feature?

**Comment by jinghancc** [comment]
don't forget to rename PR

File: pkg/oomstore/feature_test.go (hunk 1 above)
Comment (lianxmfor) [reply pkg/oomstore/feature_test.go:78]: Updated unit-test. whether the current feature is a batch or a stream is determined by the group. So here we can only determine if the group does not exist

**Comment by lianxmfor** [comment]
> don't forget to rename PR

done

**Review by YikSanChan — APPROVED** [review]
lgtm. Can you fix integration tests?

**Review by jinghancc — APPROVED** [review]
LGTM

**Comment by lianxmfor** [comment]
> lgtm. Can you fix integration tests?

the integration test is not caused by the PR. I plan to fix it in another PR.

**Comment by jinghancc** [comment]
> lgtm. Can you fix integration tests?

integration test could be ingored since we are in refactor.

**Pull request merged**